### ISSUE TITLE
propagate --group option to select_k_best_genes call

### ISSUE THREAD

**Issue by e-t-k** [jvivian/gene-outlier-detection#58]
Crashes when I use -g/--group set to anything other than tissue.
This seems to be because the call to`select_k_best_genes` doesn't pass it along despite the function accepting a `group` parameter.

https://github.com/jvivian/gene-outlier-detection/blob/fb85463d1c4c6a171db17bc6d116b1d9a255963c/gene_outlier_detection/lib.py#L114

```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/gene_outlier_detection/main.py", line 228, in <module>
    cli()
  File "/opt/conda/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/conda/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/conda/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/gene_outlier_detection/main.py", line 224, in cli
    iter_run(opts)
  File "/opt/conda/lib/python3.7/site-packages/gene_outlier_detection/main.py", line 45, in iter_run
    opts.ranks = anova_distances(opts.sample, opts.df, opts.genes, opts.group)
  File "/opt/conda/lib/python3.7/site-packages/gene_outlier_detection/lib.py", line 114, in anova_distances
    skb_genes = select_k_best_genes(df, genes, n=n_genes)
  File "/opt/conda/lib/python3.7/site-packages/gene_outlier_detection/lib.py", line 35, in select_k_best_genes
    k.fit_transform(df[genes], df[group])
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/frame.py", line 2927, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/indexes/base.py", line 2659, in get_loc
    return self._engine.get_loc(self._maybe_cast_indexer(key))
  File "pandas/_libs/index.pyx", line 108, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 132, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 1601, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 1608, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'tissue'
```

**Comment by e-t-k** [comment]
Workaround, of course, is to call your group column `tissue` regardless of what it actually is.

**Comment by jvivian** [comment]
@e-t-k — Hey Ellen, a fix for this and the `--tune` / `--save-model` arguments have all been merged and are available in the 0.13.0a release.
